packages/frontend/amp/components/TopMetaOpinion.tsx

```diff
@@ -56,6 +79,25 @@ const SeriesLink: React.SFC<{
     );
 };
 
+const BylineImage: React.SFC<{
+    tags: TagType[];
+}> = ({ tags }) => {
+    const contributor = tags.find(t => t.type === 'Contributor');
+    if (contributor && contributor.bylineImageUrl) {
+        return (
+            <amp-img
```

Comment:
Could this be an opportunity to server a `webp` image with a `fallback` as per the example [on the `amp-image` docs](https://amp.dev/documentation/components/amp-img?referrer=ampproject.org)? using the [webp param in fastly](https://docs.fastly.com/guides/imageopto-setup-use/serving-images#webp-image-support)?

Reply:
No idea about dcAMP, but dotcom image requests are `auto=webp` by default.
BTW, these contributor images are heavily templated and extremely consistent. Ratio is 6/5 and head same size, same place (centred horizontally), eyes aligned. Dotcom encirclings get this wrong BTW.

Reply:
Yes, that sounds much better. The caveat is that we do the generating image URLs in Frontend and currently only model images as a single URL in the data model.

So it might be worth adding it to the code review doc as something to think about? For example, I suspect generally, choosing the right image(s) from a CAPI URL should probably be something DCR does itself.